v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -245,6 +245,11 @@ var functions = map[string]govaluate.ExpressionFunction{
 		data := deserialization.GenerateJavaGadget(gadget, cmd, encoding)
 		return data, nil
 	},
+	// for debug purposes
+	"print_stdout": func(args ...interface{}) (interface{}, error) {
```

Comment:
if it's for debugging purposes, why not name it "debug" instead?

Reply:
I've changed it into `print_debug` 👍  @forgedhallpass